rules/coding-style/tests/Rector/Variable/VariableVariablesRector/VariableVariablesRectorTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodingStyle\Tests\Rector\Variable\VariableVariablesRector;
+
+use Iterator;
+use Rector\CodingStyle\Rector\Variable\VariableVariablesRector;
+use Rector\Core\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+final class VariableVariablesRectorTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
+    public function test(SmartFileInfo $fileInfo): void
+    {
+        $this->doTestFileInfo($fileInfo);
+    }
+
+    public function provideData(): Iterator
+    {
+        return $this->yieldFilesFromDirectory(__DIR__ . '/Fixture');
```

Comment:
Tests look good.

Could you also add 2 skipped fixture files? E.g. already migrated or ignored cases

Reply:
More than the 2 that exist already?

Reply:
Yes, maybe some completely normal variable or variable with 3 $$$ (if that's possible)